### DISCUSSION TITLE
请问，本软件用自定义的测试链接测试延迟是个问题吗

### DISCUSSION THREAD

**Discussion by cng10001** [clash-verge-rev/clash-verge-rev#1731] [General]
如题，为什么自定义测试链接都显示节点都显示error 
![image](https://github.com/user-attachments/assets/761ff28d-49dd-435b-ab56-0067882c4e50)

**Comment by iwalke** [comment]
大概率你这个机场的问题，贪便宜了吧。。。

想测也很简单，跑裸核，dashboard 没问题，用浏览器打开 twitter，也能打开，一般就是应用这边的问题。

**Reply by cng10001** [reply]
emm，怎么说呢，这个机场是便宜的，但是我觉得很好用啊，稳定，网速通畅，所以软件正常用是没问题。
你用自定义的链接测试都是正常的嘛

**Reply by cng10001** [reply]
哦 知道了 要完整的http链接
